src/components/LogIn.jsx

```diff
@@ -22,7 +23,8 @@ const LogIn = () => {
     e.preventDefault();
     const response = await login({username, password});
     if (response) {
-      console.log('success');
+      console.log(response);
+      Cookies.set('token', response.token);
```

Comment:
you are setting the cookie, but there is nothing here to trigger a re-render of `App.js` or  `onEffect` to run.

Maybe try navigating back to the home page once the cookie is set?
You can use `useNavigate` for that